lib/metro-transit.js

```diff
@@ -16,23 +16,21 @@ var MetroTransit = function () {
 MetroTransit.prototype = Object.create(AlexaSkill.prototype);
 MetroTransit.prototype.constructor = MetroTransit;
 
-MetroTransit.prototype.eventHandlers = {
-  onSessionStarted: function (sessionStartedRequest, session) {
-    console.log('MetroTransit onSessionStarted requestId: ' + sessionStartedRequest.requestId +
-        ', sessionId: ' + session.sessionId);
-    // any initialization logic goes here
-  },
+MetroTransit.prototype.eventHandlers.onSessionStarted = function (sessionStartedRequest, session) {
```

Comment:
I tried changing it to this format because I looked at the diff and noticed we were using this format in October when it worked. It's also the same format as in the sample projects.